test/unit/tracking/get-scheme-id.test.js

```diff
@@ -0,0 +1,74 @@
+const { getSchemeId } = require('../../../app/tracking/get-scheme-id')
+
+const { SFI, SFI_PILOT, LUMP_SUMS, VET_VISITS, CS, BPS, FDMR, MANUAL, ES, FC, IMPS } = require('../../../app/constants/schemes')
+const { sfi, sfip, lses, ahwr, cs, bps, fdmr, manual, genesis, glos, imps } = require('../../../app/constants/source-systems')
+
+let invoiceNumber
+
+describe('get scheme id', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks()
+  })
+
+  test('should return scheme id as 1 when SITI_SFI source system supplied', async () => {
+    const schemeId = await getSchemeId(sfi)
+    expect(schemeId).toBe(SFI)
+  })
```

Comment:
`getSchemeId` doesn't return a promise so no need for async or await on these tests.